### DISCUSSION TITLE
Make one for Soundcloud too

### DISCUSSION THREAD

**Discussion by Hedreon** [ThioJoe/YT-Spammer-Purge#774] [Ideas]
Soundcloud also has this issue with "sex spam bots" and barely anyone ever really moderates them, making it a bigger problem (Plus, I believe soundcloud only has very basic moderation tools for it, also making it a bigger problem (e.g. no detection system)) & everytime someone new or small uploads a track, the "sex spam bots" almost immediately reply with a spam comment, sometimes leaving the uploader/small artist feeling discouraged from ever really publicizing their tracks.

**Comment by Masamune3210** [comment]
Unfortunately it's not that easy. YouTube has a API specifically for interacting with videos and comments, this isn't a standard thing though. Not every site has a programmatic way of interacting with things

**Comment by ThioJoe** [comment]
Apparently soundcloud does have an API that can interact with comments. Though if I was going to make spam purge apps for other platforms, soundcloud would not exactly be top of the list tbh

**Reply by Masamune3210** [reply]
Huh, the more you learn, did not expect them to have a API for that honestly
